packages/shared/src/utils/contract/contract-types.ts

```diff
@@ -0,0 +1,77 @@
+import { ContractUpdate } from '@deriv/api-types';
```

Comment:
You don't need to review this file! I just merged this with my branch to use some of the types!